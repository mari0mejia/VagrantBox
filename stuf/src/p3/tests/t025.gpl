string a = 7 - 2;
string b = 7 - 4.2;
string d = 12.5 - 2;
string e = 12.5 - 4.2;
