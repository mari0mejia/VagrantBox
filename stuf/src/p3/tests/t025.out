gpl.cpp::main()
  input file(tests/t025.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
string a = "5"
string b = "2.800000"
string d = "10.500000"
string e = "8.300000"
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
